rpcserver.go

```diff
@@ -5542,16 +5542,26 @@ func marshallTopologyChange(topChange *routing.TopologyChange) *lnrpc.GraphTopol
 
 	nodeUpdates := make([]*lnrpc.NodeUpdate, len(topChange.NodeUpdates))
 	for i, nodeUpdate := range topChange.NodeUpdates {
+		nodeAddrs := make([]*lnrpc.NodeAddress, 0)
```

Comment:
small nit: this can be preallocated to the correct size since the final size is known, e.g. `make([]*lnrpc.NodeAddress, len(nodeUpdate.Addresses))`